decidim-elections/lib/decidim/votings/test/factories.rb

```diff
@@ -89,12 +89,19 @@
   end
 
   factory :datum, class: "Decidim::Votings::Census::Datum" do
-    document_number { Faker::IDNumber.spanish_citizen_number }
-    document_type { "DNI" }
-    birthdate { Faker::Date.birthday(min_age: 18, max_age: 65) }
+    document_number = Faker::IDNumber.spanish_citizen_number
```

Comment:
💬  you can use transient attributes for these things:

```ruby
transient do
  document_number { Faker::IDNumber.spanish_citizen_number }
  document_type { "DNI" }
  birthdate { Faker::Date.birthday(min_age: 18, max_age: 65) }
  postal_code { Faker::Address.postcode }
end
```

This attributes will not be saved in the model, but they can be used to generate values for other fields:

```ruby
hashed_in_person_data { Digest::SHA256.hexdigest([document_number, document_type, birthdate].join(".")) }
```
After that, you can set specific values for them as with regular attributes:

```ruby
create(:datum, document_number: "00000001R")
```

Reply:
Thanks for the tip! I'll update it 😀

Reply:
Nice! Didn't know that :)